### ISSUE TITLE
Wish Curation List: order wishes by date

### ISSUE THREAD

**Issue by aclaire-thd** [homedepot/infinite-wish-board#86]
Proposal to add order wishes by date to to the site. 

**Feature Name** 
order wishes by date

**Functionality**
order wishes chronologically in the wish curation list

**Notes**
I think it makes sense to have the ordering straight up chronological without a way to change sorting

**Comment by aclaire-thd** [comment]
I don't think it really matters what the date means since that will be determined by [Wish Curation List: Date for wishes](https://github.com/homedepot/infinite-wish-board/issues/85)

**Comment by jrgoodpasture** [comment]
I will pick up this story

**Comment by jrgoodpasture** [comment]
Currently date is sorted in chronological order, I will order in reverse chronological order so that the most recent wish is at the top
